### ISSUE TITLE
Re-add support for file upload button in author dashboard

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#4894]
The author dashboard shows a file upload button when revisions are requested. With the changes for versioning, the button has been placed but the file upload wizard doesn't open. Make the button open the file upload wizard and make sure the files go to the revisions grid.

**Comment by NateWr** [comment]
@ajnyga wanted to bring this to your attention since you'll be working on the author dashboard

**Comment by ajnyga** [comment]
Just had an urge to comment on this issue.

**Comment by NateWr** [comment]
A hint for when you get to this. In the author dashboard template you'll find a button like this:

```html
<pkp-button
	v-if="uploadFileUrl"
	:label="i18n.uploadFile"
	@click="openFileUpload"
></pkp-button>
```

That fires a method in the `WorkflowContainer`:

```
openFileUpload() {
	alert('todo: ' + this.uploadFileUrl);
},
```

To figure out how to open a modal from the Vue.js stuff, look at the method just above it:

```
openActivity() {
	var opts = {
		title: this.i18n.activityLog,
		url: this.editorialHistoryUrl,
		closeCallback: () => this.$refs.activityButton.$el.focus()
	};

	$(
		'<div id="' +
			$.pkp.classes.Helper.uuid() +
			'" ' +
			'class="pkp_modal pkpModalWrapper" tabIndex="-1"></div>'
	).pkpHandler('$.pkp.controllers.modal.AjaxModalHandler', opts);
},
```

**Comment by NateWr** [comment]
This is being addressed as part of #4877.
